PortMaster/pylibs/harbourmaster/config.py

```diff
@@ -94,11 +94,11 @@
     HM_DEFAULT_PORTS_DIR   = Path("/storage/roms/ports")
     HM_DEFAULT_SCRIPTS_DIR = Path("/storage/roms/ports")
 
-## Check if retrodeck.cfg exists. Chose this file/location as platform independent from were retrodeck is installed.
-elif (Path.home() / ".var/app/net.retrodeck.retrodeck/config/retrodeck/retrodeck.cfg").is_file():
-    retrodeck_roms_path = retrodeck_roms_path = (Path.home() / ".var/app/net.retrodeck.retrodeck/config/retrodeck/retrodeck.cfg").read_text()
+## Check if retrodeck.sh exists. Chose this file/location as platform independent from were retrodeck is installed.
+elif ("/var/app/bin/retrodeck.sh").is_file():
```

Comment:
Small typo, needs to be a Path object:
```suggestion
elif Path("/var/app/bin/retrodeck.sh").is_file():
```